packages/launcher/src/session.ts

```diff
@@ -0,0 +1,26 @@
+import type { SessionOptions } from "@treasure-dev/tdk-core";
+import axios from "axios";
+import { isUsingTreasureLauncher } from "./utils";
+
+export function startUserSessionViaLauncher({
+  backendWallet,
+  approvedTargets,
+  nativeTokenLimitPerTransaction,
+  sessionDurationSec,
+  sessionMinDurationLeftSec,
+}: SessionOptions): Promise<void> {
+  if (!isUsingTreasureLauncher()) {
+    return Promise.reject(
+      new Error(
+        "startUserSessionViaLauncher can only be used with Treasure Launcher",
+      ),
+    );
+  }
+  return axios.post("http://localhost:16001/tdk-start-session", {
```

Comment:
Is there any benefit to using `axios` over native `fetch`? I'd opt to add as few dependencies as possible to our own packages

Reply:
Makes sense, will make that change